llvm/test/CodeGen/AMDGPU/GlobalISel/crash-stack-address-O0.ll

```diff
@@ -25,4 +25,4 @@ entry:
 }
 
 !llvm.module.flags = !{!0}
-!0 = !{i32 1, !"amdgpu_code_object_version", i32 500}
+!0 = !{i32 1, !"amdhsa_code_object_version", i32 500}
```

Comment:
Can we remove explicit mention of COV5 module flag in all these test files given that the current default is COV5?
Or, may be a separate patch would be better for that.

Reply:
Separate would be better

Reply:
Sure, I'll make a PR for that too after this lands.